library/core/src/num/f32.rs

```diff
@@ -18,6 +18,14 @@ use crate::mem;
 use crate::num::FpCategory;
 use crate::panic::const_assert;
 
+use safety::requires;
+
+#[cfg(kani)]
+use crate::kani;
+
+#[cfg(kani)]
```

Comment:
Will this work? 

```suggestion
#[allow(unused_imports)]
```

Reply:
Yes. Is this better than `#[cfg(kani)]`? I can update it now.

Reply:
Yes, that will make other tools integration easier

Reply:
Fixed